src/auro-pane.js

```diff
@@ -35,18 +71,78 @@ class AuroPane extends LitElement {
     `;
   }
 
-  // function that renders the HTML and CSS into  the scope of the component
+  /**
+   * Internal method to parse the date attribute
+   * @private
+   * @returns {Object} object containing day, date, and month
+   */
+  parseDateString() {
+    const dateFormatLength = 10;
+
+    if (this.date && this.date.length === dateFormatLength) {
+      // Using substring instead of date parsing due to browser inconsistencies
+      const year = this.date.substring(0, 4),
+        month = this.date.substring(5, 7),
+        date = this.date.substring(8),
+        parsedDate = new Date(year, month - 1, date);
+
+      return {
+        day: this.weekdays[parsedDate.getUTCDay()],
+        date: parsedDate.getUTCDate(),
+        month: this.months[parsedDate.getUTCMonth()]
+      };
+    }
+
+    return {};
+  }
+
+  /**
+   * Internal method that determines how to display the price attribute
+   * @private
+   * @returns {TemplateResult} price HTML
+   */
+  getPrice() {
+    if (this.price) {
+      const priceClasses = {
+        'price': true,
+        'price--long': this.price.length > 6
+      };
+
+      return html`<div class="${classMap(priceClasses)}">${this.disabled ? "––" : this.price}</div>`
+    }
+
+    return html``;
+  }
+
+  /**
+   * Programatically focuses the component
+   * @return {void}
+   */
+  focus() {
+    this.renderRoot.querySelector('button').focus();
+  }
+
   render() {
+    const buttonClasses = {
+      'auro-pane': true,
+      'auro-pane--selected': this.selected,
```

Comment:
@blackfalcon, any idea on how to indicate the "selected" state to screen readers? Currently it's only a visual indication which isn't great. Maybe some screen reader only text to read out "selected"? I was also looking into [aria-selected](https://www.w3.org/TR/wai-aria-1.1/#aria-selected) but it doesn't look like that's valid for button roles.